change/@fluentui-react-charting-cae42a98-eff0-4072-a40f-8a0d2add5253.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33406&commitId=70f38511021d13ddfadfbc9fae0921a61b289ccd&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Gap space 25.chromium.png|2309| Changed|
|Callout.Left bottom edge.chromium.png|3022| Changed|
|Callout.No beak.chromium.png|2306| Changed|
|Callout.Rendering callout attached to a rectangle.chromium.png|1493| Changed|
|Callout.Top center.chromium.png|2178| Changed|

</details>


  <details><summary><b>Keytip</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Keytip.Offset.chromium.png|121| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>


  <details><summary><b>react-charting-VerticalBarChart</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Basic Secondary Y Axis RTL.chromium.png|474| Changed|
|react-charting-VerticalBarChart.Basic Secondary Y Axis Dark Mode.chromium.png|495| Changed|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis.chromium.png|474| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>